src/NuGet.Core/NuGet.Configuration/PackageSourceMapping/PackageSourceMapping.cs

```diff
@@ -20,34 +22,39 @@ public class PackageSourceMapping
         /// </summary>
         internal IReadOnlyDictionary<string, IReadOnlyList<string>> Patterns { get; }
 
-        private Lazy<SearchTree> SearchTree { get; }
+        private Lazy<SearchTree>? SearchTree { get; }
```

Comment:
Also note that `SearchTree` does not meet our naming style. Private, non-static fields should be `_camelCased`, not `PascalCased`.  Isn't VS putting a squiggle under the name?

Reply:
> Also note that SearchTree does not meet our naming style. Private, non-static fields should be _camelCased, not PascalCased. 

According to our coding guideline # 3, this only applies to fields. `SearchTree` is a private property. I'm not sure why the creator of the property made it a property, but nothing has changed in this PR.

> Isn't VS putting a squiggle under the name?
Nope
![image](https://github.com/NuGet/NuGet.Client/assets/49205731/067f7536-77cc-430b-b290-18332b40251d)